docker.md

```diff
@@ -0,0 +1,18 @@
+
+## Building The Site Locally By Docker
+
+```bash
+repath=~/git/project/ # The folder path of repository bitcoinops.github.io
+
+cd ${repath}
+
+git clone https://github.com/bitcoinops/bitcoinops.github.io
+
+docker run -d --name bitcoinops -p 4000:4000 -v ${repath}/bitcoinops.github.io:/root/bitcoinops.github.io -w /root/bitcoinops.github.io ruby:2.6.4-stretch /bin/bash -c "bundle install && make preview"
+
+# open url 127.0.0.1:4000 in browser
```

Comment:
This is not immediately available in the browser while the bundle install and make commands are running, so perhaps we can note the delay or provide a way for users to check on the build process (docker logs <container ID> ?) ?

Reply:
Yeah, we can check whether is it deployed success by ``docker logs -f bitcoinops``, it deploys success if a user sees the log 'Server running'. I will update the file docker.md.
<img width="626" alt="image" src="https://user-images.githubusercontent.com/41293784/220496232-80a683a6-3b47-4ea3-8c25-20e5e7f383b9.png">